app/packages/_components/list_flank/list_flank.coffee

```diff
@@ -15,16 +27,37 @@ Template.list_flank.events
     else
       data.selected.set itemId
 
+    _allItemsSelected = instance.allItemsSelected
+    if allItemsSelected(instance)
```

Comment:
Ahh good point.  I thought that function was returning a count, but after looking at it more closely i see it's actually returning a bool.  Since it's returning a bool I think the function name makes sense.